core/common/src/main/java/alluxio/master/file/meta/PersistenceState.java

```diff
@@ -21,5 +21,42 @@ public enum PersistenceState {
   NOT_PERSISTED, // file not persisted in the under FS
   TO_BE_PERSISTED, // the file is to be persisted in the under FS
   PERSISTED, // the file is persisted in the under FS
-  LOST // the file is lost but not persisted in the under FS
+  LOST; // the file is lost but not persisted in the under FS
+
+  /**
+   * @return the persistence state as a protobuf object
+   */
+  public alluxio.grpc.PersistenceState toProto() {
```

Comment:
Is it possible to remove the `alluxio.master.file.meta.PersistenceState` altogether and rely solely on the `alluxio.grpc.PersistenceState`? To reduce complexity and this translation layer?